afm/filesystems/vault.py

```diff
@@ -15,13 +15,15 @@ def get_jwt_from_file(file_name):
 def vault_jwt_auth(jwt, vault_address, vault_path, role):
     """Authenticate against Vault using a JWT token (i.e., k8s sa token)"""
     full_auth_path = vault_address + vault_path
-    logger.debug("full_auth_path = %s", str(full_auth_path))
+    logger.debug('authenticating against vault using a JWT token',
+        extra={'full_auth_path': str(full_auth_path)})
     json = {"jwt": jwt, "role": role}
     response = requests.post(full_auth_path, json=json)
     if response.status_code == 200:
         return response.json()
-    logger.error("Got error code %d from Vault authentication", response.status_code)
-    logger.error("Error response: %s", str(response.json()))
+    logger.error("vault authentication failed",
+        extra={"status_code": str(response.status_code),
```

Comment:
The key should be error, so that we match the keys that are generated by zerolog.  See examples here: https://github.com/rs/zerolog/blob/master/log_example_test.go.  I would suggest that the standard keys be defined in constants in logging.py for use by all.

Reply:
adding 'status_code', 'error', and 'AssetId' as constants.

Reply:
removing 'status_code' constant. The status code will be part of the error message.